src/components/Editor/index.js

```diff
@@ -173,7 +183,20 @@ const Editor = React.createClass({
       menuOptions.push(watchExpressionLabel);
     }
 
+    if (isEnabled("copySource")) {
+      menuOptions.push(copySourceUrl);
+    }
+
+    const showSourceMenuItem = {
+      id: "node-menu-show-source",
+      label: "Show source",
+      accesskey: "s",
+      disabled: false,
+      click: () => showSource(tab)
+    };
+
```

Comment:
I just realized, does it make sense to have showSource in the Editor? We'd already be on the correct document when the context menu comes up (assuming showSource literally just shows the source of the file)